ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue

```diff
@@ -363,9 +362,9 @@ export default defineComponent({
         maxLength(70)
       ),
       phoneNumberRules: customRules(
-        required('Enter a phone number'),
-        maxLength(15)
-      )
+        isPhone(14, null)
```

Comment:
In your rule you've defined `customMsg` with a default, so you don't need to pass in `null` here.

Reply:
Dima caught that one as welll.  I have made the change..